src/phonebook/Record.java

```diff
@@ -0,0 +1,11 @@
+package phonebook;
+
+public class Record {
+    String name = null;
```

Comment:
access modificatory missing, and, also, null is default value for string, so you don't need to set it